### ISSUE TITLE
Staging Review finding: Incorrect heading structure

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#101619]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Disability Benefits Experience Team 1
**Product:** 526ez
**Feature:** Add 5103 Language and Claim Certification to 526ez

## Findings details
**VA.gov Experience Standard - issue:** [User can determine next steps in a flow.](https://design.va.gov/about/experience-standards/#user-can-determine-next-steps-in-a-flow)
**VA.gov Experience Standard - category:** [Findability](https://design.va.gov/about/experience-standards/#category-findability)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

On [/supporting-evidence/orientation](https://staging.va.gov/disability/file-disability-claim-form-21-526ez/supporting-evidence/orientation), there's an info alert with H2 heading "Notice of evidence needed." For the forms library, the step indicator for each page ("3 of 5 Supporting Evidence") is also an H2 heading.

Screen reader users often use headings as wayfinding tools, where a user will skip heading-to-heading listening for the section they're interested in. At each heading, the heading level is announced, which provides information about the structure of the page --- big sections, sub-sections, sub-sub-sections, etc.

As I read the content of this page, the "Notice of evidence needed" alert is a sub-section of the "Supporting Evidence" section. Having both as H2s is misleading to users.

<details><summary>Link, screenshot or steps to recreate</summary>

![Supporting Evidence orientation page, with a blue info alert with heading "Notice of evidence needed."](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/82160-Brian-1.png?raw=true)

</details>

## Recommended action

Update the "Notice of evidence needed" heading to be an H3.

## References

- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
